.env.development

```
@@ -4,3 +4,6 @@ LOGRAGE_IN_DEVELOPMENT=true
 # SPOTIFY API CREDENTIALS
 SPOTIFY_CLIENT_ID=
 SPOTIFY_CLIENT_SECRET=
+
+# REDIS URL
+REDIS_URL="redis://localhost:6379" # default connection for localhost
```

Il manque l'id de la db

```suggestion
REDIS_URL="redis://localhost:6379/0" # default connection for localhost
```